tests/external_botocore/test_bedrock_chat_completion.py

```diff
@@ -144,6 +147,15 @@ def expected_events_no_content(expected_events):
     return events
 
 
+@pytest.fixture(scope="module")
+def expected_events_with_token_count(expected_events):
```

Comment:
I'm going to suggest copying something Lalleh did in her PR where she made a generic function in conftest that goes through and modifies any expected event input generically. It looks like you did this in openai as well. I think this is overall simpler than trying to utilize a fixture as the expected events are not named the same in each test.
So in conftest put the following:
```
def add_token_count_to_message_events(expected_events):
    events = copy.deepcopy(expected_events)
    for event in events:
        if event[0]["type"] == "LlmChatCompletionMessage":
            event[1]["token_count"] = 105
    return events
```

And then you can use it everywhere like:
```
@validate_custom_events(add_token_count_to_message_events(expected_events_with_llm_metadata))
```

Reply:
It looks like none of the expected event test cases in Bedrock follow this approach where it's passed to the event modifying function. It's definitely cleaner as you said so I have updated the token count events to follow this format and will fix the rest of them (no metadata, no content, etc.) in a subsequent cleanup PR.